elements/bulbs-video/components/revealed.js

```diff
@@ -256,11 +260,16 @@ export default class Revealed extends React.Component {
     GoogleAnalytics.init(this.player, videoMeta.gaTrackerAction);
     Comscore.init(this.player, global.BULBS_ELEMENTS_COMSCORE_ID, videoMeta.player_options.comscore.metadata);
 
+    this.player.on('complete', this.forwardJWEvent);
```

Comment:
@collin, which part of the PR dictates that it only forwards the event when content ends and not ads?